proto/sentry_protos/snuba/v1/endpoint_find_traces.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message EventFilter {
+  string trace_item_name = 1;
+  TraceItemFilter filter = 2;
+}
+
+message AndTraceFilter {
+  repeated EventFilter filters = 1;
+}
+
+message OrTraceFilter {
+  repeated EventFilter filters = 1;
+}
+
+message NotTraceFilter {
+  repeated EventFilter filters = 1;
+}
```

Comment:
these make it so you can't OR/AND anything except EventFilters. You probably want 

```
message AndTraceFilter {
  repeated TraceFilter filters = 1;
}
```

Otherwise you will only be able to express oneof (AndTraceFilter, OrTraceFilter, NotTraceFilter, EventFilter)

Reply:
So I actually collapsed Trace/EventFilter, I think that fixes this though.

Reply:
I don't think I understand, how would you be able to express conditions like

`(A OR B) AND (C OR D)`

With this structure? Or is that purposefully left out?

Reply:
```
AndTraceFilter(
    OrTraceFilter(
        TraceFilter(A), 
        TraceFilter(B),
    ),
    OrTraceFilter(
        TraceFilter(C),
        TraceFilter(D),
    ),
)
```

Reply:
I think I have this fixed now.